src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs

```diff
@@ -460,7 +473,13 @@ public string InputSymbol
                     }
                     else
                     {
-                        Parameter = new TypedParameter(name, type, defaultValue, null, comment);
+                        Error("This input symbol is not currently valid, " +
+                              "and cannot be saved until it is fixed. " +
+                              "Valid inputs take the form:\n\n" +
+                              "name : type = defaultValue\n\n" +
+                              "The input name should be a valid variable name, without spaces. An input type and default value are optional."); //TODO: move to resx");
```

Comment:
Same for the error message, please put it as resource string so we have localization of it